backend/models.py

```diff
@@ -17,12 +17,13 @@ class UserMetrics(EmbeddedDocument):
     works_visited = IntField(default=0)
     works_found = IntField(default=0)
     cities_visited = IntField(default=0)
-    posts_written = IntField(default=0)
+    posts_written = IntField(default=0) # not entirely sure what this represents, might wanna change it to works_created
```

Comment:
I think works_added could be a better name, so it doesn't imply the user created the art they are adding.